tests/cases/compiler/contravariantOnlyInferenceFromAnnotatedFunctionJs.ts

```diff
@@ -0,0 +1,30 @@
+// @strict: true
```

Comment:
this reuses the existing test case for a TS file: https://github.com/microsoft/TypeScript/pull/52609/files#diff-85ed3085ca9f5d4c893196581ea0785e5d6a5aca5ae449afe499fb4d4d159728